testDataSetup/mongo/mongo.go

```diff
@@ -85,26 +65,17 @@ func TeardownMany(d *ManyDocs) error {
 	return nil
 }
 
-// Setup is a way of loading in an individual document into a mongo instance
-func Setup(database, collection, key, value string, update bson.M) error {
-	s := session.Copy()
-	defer s.Close()
-
-	if _, err := s.DB(database).C(collection).Upsert(bson.M{key: value}, update); err != nil {
-		log.ErrorC("mongodb datastore error", err, nil)
-		return err
+// Setup is a way of loading any number of documents into a mongo instance
+func Setup(d ...*Doc) error {
+	if err := Teardown(d...); err != nil {
+		log.ErrorC("Unable to remove test data from mongo db", err, nil)
+		os.Exit(1)
```

Comment:
Should this return an error and let the test decide to handle it?

Reply:
Fair point. I just copied a Teardown call from one of the tests, will return err instead